site.yml

```diff
@@ -166,10 +183,10 @@
   roles:
     - role: openstack-database
       database_name: glance
-      tags: ['openstack', 'glance', 'control', 'ceph']
-
+      tags: ['openstack', 'glance', 'control']
     - role: glance
-      tags: ['openstack', 'glance', 'control', 'ceph']
```

Comment:
probably a rebase bug?       not sure why glance role would have a tag for `ceph` ?